plugins/installer.plugin

```diff
@@ -152,6 +152,11 @@ esac
 . /usr/share/initramfs-tools/hook-functions
 
 EOF
+    if ! echo $ARCH | grep -q arm; then
```

Comment:
how about aarch64 [ARM64]

Reply:
Is arm64 called aarch64 here? If it is, good point :D